### DISCUSSION TITLE
Receiving 404 error

### DISCUSSION THREAD

**Discussion by sarakr4** [CollectionBuilder/collectionbuilder.github.io#68] [Q&A] [answered]
Hi! I need help figuring out why my collection isn't being published. I've double checked my config file and deploy from the main branch. At this point I could use a fresh set of eyes to see if I've made any errors. Any help is appreciated, thanks in advance!

https://github.com/sarakr4/ldshymnals

**Comment by evanwill** [comment, marked as answer]
@sarakr4 you are close! All the configuration and files are correct, however, the build is breaking because of an incorrect include in your "about.md"--so your site isn't being built, thus you are getting the 404 still!

To fix the issue, edit your ["pages/about.md"](https://github.com/sarakr4/ldshymnals/blob/main/pages/about.md), take a look at line 27 and 31 you have an include like:

`{% include feature/image-external.html src="https://www.churchofjesuschrist.org/media/image/emma-hale-smith-lee-richards-9b757db?lang=eng" width="50" caption="Emma Hale Smith"%}`

This is trying to use the include "_includes/image-external.html", which doesn't actually exist! (I believe that *was* an include in an older version of the template, so maybe you got that from some out of date docs?)
You will have to replace the two "feature/image-external.html" includes with the "feature/image.html" include. The options are similar. The one above should be something like:

`{% include feature/image.html objectid="https://assets.ldscdn.org/b6/b0/b6b0a878090d000d5867ab7e26b05538f7f23408/emma_hale_smith_lee_richards.jpeg" width="50" caption="Emma Hale Smith" alt="painted portrait of a women in a white shawl" %}`

Notice I changed "feature/image-external.html" to "feature/image.html"; "src" to "objectid"; updated the link you were using to one that specifically referenced an image to display (the other link was to an descriptive page, so can't be displayed as an image); and added an "alt" text.

Your second one would be updated something like: 

`{% include feature/image.html objectid="1905-sacred-hymns-and-spiritual-songs-for-the-church-of-jesus-christ-of-latter-day-saints" width="50" %}`

Since it is an internal image from your collection, you can use the "objectid" of the item from your metadata to call the information in.

Good luck!

**Reply by sarakr4** [reply]
It's fixed! Thank you a million times!!!
